gce/survival-training/dummy/train.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import copy
+import glob
+import json
+import os
+import random
+
+
+def generate_trainer(hyperparameters):
+  """Generates a callable which performs a single step of training when called.
+
+  Args:
+    1. hyperparameters - hyperparameters to train with.
+
+  Returns:
+    trainer callable
+  """
+  def _trainer():
+    return random.random()
+
+  return _trainer
+
+
+def runner(
+    trainer_initializer,
+    job_dir,
+    train_steps,
+    checkpoint_steps,
+    hyperparameters
+    ):
+  """Runs a training job.
+
+  Args:
+    trainer_initializer: Function which accepts hyperparameter dictionary as its
+    only argument and returns a callable representing a single step of training.
+    job_dir: Directory in which checkpoints should be stored.
+    train_steps: Total number of steps for which training should be performed.
+    checkpoint_steps: Training steps between checkpoints.
```

Comment:
it might help if you use the same names as in tensorflow.  For example this might be something like `n_checkpoint_steps`.